src/lustre/lustreAstDependencies.ml

```diff
@@ -806,6 +812,7 @@ let rec mk_graph_expr2: node_summary -> LA.expr -> (dependency_analysis_data lis
              empty_dependency_analysis_data
              (List.concat gs)]
 
+  | LA.ChooseOp (_, _, e) -> mk_graph_expr2 m e 
```

Comment:
This should be handled as a node call.

Reply:
Still pending based on if we commit to moving choose op desugaring earlier in pipeline